frontend/integration-tests/views/monitoring.view.ts

```diff
@@ -0,0 +1,43 @@
+import { $, $$, browser, by, ExpectedConditions as until } from 'protractor';
+
+import * as crudView from '../views/crud.view';
+
+export const wait = async(condition) => await browser.wait(condition, 15000);
+
+// List pages
+export const listPageHeading = $('.co-m-pane__heading');
+export const firstListLink = $$('.co-resource-list__item a.co-resource-link__resource-name').first();
+export const createButton = $('.co-m-pane__filter-bar-group button');
+export const rowMenuButton = row => row.$('.co-kebab__button');
+
+// Details pages
+export const expiredSilenceIcon = $('.co-m-pane__details .fa-ban');
+export const ruleLink = $('.co-m-pane__details .co-resource-link__resource-name');
+export const silenceComment = $$('.co-m-pane__details dd').get(-2);
+export const firstAlertsListLink = $$('.co-resource-list__item a.co-resource-link').first();
+
+export const testDetailsPage = async(subTitle, alertName, expectLabel = true) => {
+  const heading = $('.co-m-nav-title .co-m-pane__name');
+  await wait(until.presenceOf(heading));
+  expect(heading.getText()).toContain(alertName);
```

Comment:
Do not have `expect()` calls in a view object function.

Reply:
I converted `testDetailsPage()` into a helper in `tests/monitoring.scenario.ts`.